src/scheduler/mod.rs

```diff
@@ -12,6 +11,8 @@ mod round_robin;
 pub(crate) mod metrics;
 pub(crate) mod serialization;
 
+pub use crate::runtime::task::TaskId;
```

Comment:
Why make this pub?

Reply:
It's to be able to implement the `Scheduler` trait, which is also pub, in client code (in this case, in a unit test). I'm actually surprised the compiler didn't complain about it before.